File: Python/8py 3/shark.py
"""
|--------------------------------------------------------------------------
| Holiday VI - Shark Pontoon
|--------------------------------------------------------------------------
|
"""

def shark(pd, sd, ys, ss, d):
   return 'Alive!' if pd / ys < sd / (ss / 2 if d else ss) else 'Shark Bait!'


def shark(d1, d2, v1, v2, x):
   return "Alive!" if d1 / v1 < d2 / v2 * (x + 1) else "Shark Bait!"


def shark(s,h,a,r,k):
   return ["Shark Bait!","Alive!"][h/r*(k+1)>s/a]


print(shark(12, 50, 4, 8, True), "Alive!")
print(shark(7, 55, 4, 16, True), "Alive!")
print(shark(24, 0, 4, 8, True), "Shark Bait!")

File: PHP/8php 3/shark.php
<?php
/*
|--------------------------------------------------------------------------
| Holiday VI - Shark Pontoon
|--------------------------------------------------------------------------
|
*/

function shark(int $pd, int $sd, int $ys, int $ss, bool $d): string {
   return $pd / $ys < $sd / ($d ? $ss / 2 : $ss) ? 'Alive!' : 'Shark Bait!';
}


function shark(int $pontoonDistance, int $sharkDistance, int $youSpeed, int $sharkSpeed, bool $dolphin): string {
   return ($pontoonDistance / $youSpeed < $sharkDistance / $sharkSpeed << $dolphin) ? "Alive!" : "Shark Bait!";  
}


function shark(int $pD, int $sD, int $yS, int $sS, bool $d): string {
   return ($pD / $yS) < ($sD / $sS) * ($d ? 2 : 1) ? "Alive!" : "Shark Bait!";
}


print_r(shark(12, 50, 4, 8, true) . " Alive!
");
print_r(shark(7, 55, 4, 16, true) . " Alive!
");
print_r(shark(24, 0, 4, 8, true) . " Shark Bait!
");